content/outcomes/cloud-native-development/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cloud Native Development
+weight: -1
+layout: single
+description: "Learn the basics of cloud native development with Spring. This developer pre-engagement guide helps you to get the most of an engagement following the Tanzu Labs methodology."
+team:
+ - VMware Tanzu Labs
+---
+ 
+## Why is this important?
+
+This learning path will provide you with foundational knowledge around modern, cloud-native software development with Spring and Cloud-Foundry. It has been conceived as pre-work to get the most of an application-related software development engagement with Tanzu Labs. 
```

Comment:
> It has been conceived as pre-work to get the most of an application-related software development engagement with Tanzu Labs. 

Is this guide intended to be specifically for consultants working on Tanzu Labs projects, or a general guide and overview of Cloud Native development? 

In general we should publish content without assuming anyone is a consultant on an engagement. For example, the Modern Application Development outcome makes only the briefest mention the the content is valuable if someone is going to work with Tanzu Labs.

I suggest we adjust the content wherever consulting, engagements, clients, etc. are discussed.

Reply:
Alright! I reworded this page also following the feedback of @zmb3  Now it should be better. :-)